### PR TITLE
Add ssl context container

### PR DESCRIPTION
**Pull request by maximilianpilz**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have updated the [changelog](../CHANGELOG.md) accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

**Review by leon1995 — CHANGES_REQUESTED** [review]

File: src/sdc11073/certloader.py
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/certloader.py:1]: I do not agree. It is a top level functionality needed by user. And the name is fine, it does what it says.

File: src/sdc11073/certloader.py [hunk 1]
```diff
@@ -1,23 +1,30 @@
...
-                               ca_public_key='cacert.pem',
-                               cyphers_file=None,
-                               ssl_passwd=None) -> ssl.SSLContext:
+@dataclasses.dataclass
+class SSLContextContainer:
+    client_context: ssl.SSLContext
+    server_context: ssl.SSLContext
+
+
+def mk_ssl_contexts_from_folder(ca_folder,
```
[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/certloader.py:12]: why???

**Review by deichmab-draeger — DISMISSED** [review]

File: src/sdc11073/certloader.py
[the remark this answers is not in the document]
Comment (leon1995) [reply src/sdc11073/certloader.py:1]: the methods in the file are used for ssl context

File: src/sdc11073/certloader.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (leon1995) [reply src/sdc11073/certloader.py:12]: because it can be optimized and reduced

[the remark this answers is not in the document]
Comment (deichmab-draeger) [reply src/sdc11073/certloader.py:12]: It makes / creates something, therefore i think the name is fine. Please suggest something better. 
I doubt that a review on this level makes any sense, btw.

**Review by deichmab-draeger — APPROVED** [review]

**Pull request merged**